apps/ledger-live-mobile/src/react-native-hw-transport-ble/index.ts

```diff
@@ -36,6 +50,15 @@ const transport = Config.MOCK
           serviceUUIDs,
         };
       },
-    })
-  : DeviceManagementKitTransport;
-export default transport;
+    });
+  } else {
+    // when not in MOCK mode, return DeviceManagementKitTransport if enabled,
+    // otherwise BleTransport. Asserting the type of DeviceManagementKitTransport
+    // to match BleTransport.
+    return isLDMKEnabled
+      ? (DeviceManagementKitTransport as unknown as typeof BleTransport)
```

Comment:
[WOULD] I don't think it's a good idea to force a type to this transport

Reply:
[SHOULD] +1
this is not a good idea, and it will already break, as you can see as `observeState` (called in another file) is not defined on `DeviceManagementKitTransport`